homeassistant/components/http.py

```diff
@@ -526,3 +587,47 @@ def handle(request):
         return web.Response(body=result, status=status_code)
 
     return handle
+
+
+class IpBan(object):
+    """Represents banned IP address."""
+
+    def __init__(self, ip_ban: str, banned_at: datetime=None) -> None:
+        """Initializing Ip Ban object."""
+        self.ip_address = ip_address(ip_ban)
+        self.banned_at = banned_at
+        if self.banned_at is None:
+            self.banned_at = datetime.utcnow()
+
+
+def load_ip_bans_config(path: str):
+    """Loading list of banned IPs from config file."""
+    ip_list = []
+    ip_schema = vol.Schema({
+        vol.Optional('banned_at'): vol.Any(None, cv.exact_time)
+    })
+
+    try:
+        list_ = load_yaml_config_file(path)
+        for ip_ban in list_:
+            try:
+                ban = ip_schema(list_[ip_ban])
+                ban['ip_ban'] = ip_address(ip_ban)
+                ip_list.append(IpBan(**ban))
+            except vol.Invalid:
+                continue
+
+    except(HomeAssistantError, FileNotFoundError):
+        return []
```

Comment:
Ok, adding check prior to this. For FileNotFount it doesn't make sense to report -- it just means that no bans have being applied